### ISSUE TITLE
Option to save images to a subdirectory within your downloads directory

### ISSUE THREAD

**Issue by mfsoup** [gurumukhi/youtube-screenshot#37]
The ability to save images to subdirectores of the download directory would allow screenshots to be automatically uploaded using Sharex's watch folder feature.

**Comment by joggee-fr** [comment]
I am not aware of a way to download in a custom directory or a subdirectory of the download directory for a web extension.
It seems we can at least let the user choose the destination through a file chooser using [downloads.download()](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/downloads/download). But the file chooser will be displayed each time we have to save an image.
An other solution may be to add a configurable prefix to the image name. Can this feature be a solution for your needs with Sharex?

**Comment by mfsoup** [comment]
Yes, any prefix would allow Sharex to pick up just these files. Unfortunately if saving to subdirectories isn't possible I probably wouldn't use the feature due to cluttering the downloads folder.
